src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -215,6 +215,18 @@ void GcsServer::InitGcsJobManager(const GcsInitData &gcs_init_data) {
   gcs_job_manager_ = std::make_unique<GcsJobManager>(gcs_table_storage_, gcs_pub_sub_,
                                                      *runtime_env_manager_);
   gcs_job_manager_->Initialize(gcs_init_data);
+
+  // Init job distribution.
+  if (RayConfig::instance().gcs_task_scheduling_enabled()) {
```

Comment:
Can we rename this `gcs_actor_scheduling_enabled`?

Reply:
We name it this way because this gcs-based scheduling is also used by placement groups.

Reply:
Semantically, I think actor scheduling is still much less confusing. Tasks are not very commonly used with placement groups, so 90% of the use cases here are only affecting actors.